.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2.0
+
+jobs:
+  tests:
+    docker:
+    - image: circleci/python:3.8.7
+      environment:
+        POSTGRES_USER: osmcha
+        POSTGRES_PASSWORD: osmcha
+        POSTGRES_DB: osmcha
+        COVERALLS_REPO_TOKEN: $COVERALLS_REPO_TOKEN
+    - image: circleci/postgres:13-postgis
+      environment:
+        POSTGRES_USER: osmcha
+        POSTGRES_PASSWORD: osmcha
+        POSTGRES_DB: osmcha
+    steps:
+    - checkout
+    - restore_cache:
+        key: deps-{{ .Branch }}-{{ checksum "requirements/test.txt" }}
```

Comment:
As we have more than one requirement file, we would need to verify the checksum of all files to have a correct key, maybe using something like:
`cat requirements/local.txt requirements/base.txt requirements/test.txt requirements/production.txt | md5sum -t`

Reply:
Following steps from [circleci caching docs](https://circleci.com/docs/2.0/caching/) I'll add multiple files in same command. Could be a problem if you plan to create more requirements files.

`deps-{{ .Branch }}-{{ checksum "requirements/local.txt" }}-{{ checksum "requirements/base.txt" }}-{{ checksum "requirements/test.txt" }}-{{ checksum "requirements/production.txt" }}`

Another way is to create a checksum command and run over requirements files as mentioned before.